js/symb/taxa.taxonomyloader.js

```diff
@@ -195,3 +195,32 @@ function acceptanceChanged(f){
 	if(accStatusObj[0].checked) document.getElementById("accdiv").style.display = "none";
 	else document.getElementById("accdiv").style.display = "block";
 }
+
+// listener for taxon rank
+
+document.getElementById('rankid').addEventListener('change', function () {
+	var selectedValue = this.value; // Get the chosen value
+	$rankId = selectedValue;
+
+	var div1 = document.getElementById('div1hide');
+	var div2 = document.getElementById('div2hide');
+	var label = document.getElementById('unitind1label');
+
+	if (selectedValue > 150) {
+		div1.style.display = 'block';
+		div2.style.display = 'block';
+	} else {
+		div1.style.display = 'none';
+		div2.style.display = 'none';
+	}
+	if (selectedValue <= 180) {
+		// Get the name of selected option
+		var selectedOption = this.options[this.selectedIndex];
```

Comment:
I'm not sure what is happening here, but cool haha!

Reply:
I'm just assigning the selected option to selectedOption based on the index. Once user chooses an option from selector its index is stored in selectedIndex property. I just thought it would be easier than searching by value.